web/app/css/breadcrumb-header.css

```diff
@@ -0,0 +1,17 @@
+@import 'styles.css';
+
+.ant-layout-header {
```

Comment:
some visual feedback: 
I think this would look more like a part of the header if it were 84px tall ( same height as the linkerd logo container).